### PR TITLE
dev-proc,msi_z690: hcl process improvements

### PR DESCRIPTION
**Pull request by pietrushnic**
Signed-off-by: Piotr Król <piotr.krol@3mdeb.com>

### CODE REVIEW

File: docs/dev-proc/hcl-maintainer.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+In vim:
+
+- ++shift+v++ - to enter visual mode and mark whole range of entries.
+- type `:` and `sort` to run vim sort function on marked range.
+
+### Github report
+
+<!--
+Current template for adding HCL report over Github issue is way too complex.
+Nobody will wast time doing that.
```
Comment (miczyg1) [inline docs/dev-proc/hcl-maintainer.md:26]: So how it should look like to not cause trouble for us? To not cause ping-pongs to get missing data? If somebody does not want to contribute directly, then should cope with some inconvenience of more complex reporting. It's for everybody's benefit, not one's convenience

Comment (pietrushnic) [reply docs/dev-proc/hcl-maintainer.md:26]: > So how it should look like to not cause trouble for us?

Let's think about community, users and customers first.

Tactics of complex HCL and coping with some inconvenience gives us no results. Number of HCL reports provided over GH issues template is zero. So we have clear evidence in number that it doesn't work.

Goal of HCL is to improve users life by providing information about compatibility. We sitting on ton of HCL reports (almost 3GB), most information from those reports are not exposed to users, all those reports were provided using DTS.

HCL reports have to be trivial.

We have to change it. I really believe linux-hardware.org and bsd-hardware.info is the way to go further.

Meanwhile we have to deal with still growing number of HCL reports which were sent to us and are of no use to community.

In case of HCL reports for debugging purposes this is different story and is out of scope of what I'm proposing here. DTS should have good enough dump to provide enough debugging information for diagnostics, also maybe hw-probe could would be good enough.

Comment (miczyg1) [reply docs/dev-proc/hcl-maintainer.md:26]: > Tactics of complex HCL and coping with some inconvenience gives us no results. Number of HCL reports provided over GH issues template is zero. So we have clear evidence in number that it doesn't work.
> 
> Goal of HCL is to improve users life by providing information about compatibility. We sitting on ton of HCL reports (almost 3GB), most information from those reports are not exposed to users, all those reports were provided using DTS.
> 
> HCL reports have to be trivial.
> 
> We have to change it. I really believe linux-hardware.org and bsd-hardware.info is the way to go further.
> 
> Meanwhile we have to deal with still growing number of HCL reports which were sent to us and are of no use to community.

Well, I think community is learning slowly. As we kindly ask them for HCL reports each time they ask about issues, more and more HCLs are being attached to issues. See also this one: https://github.com/Dasharo/dasharo-issues/issues/409#issuecomment-1511217858

Aren't we jumping to conclusions too fast? And there I thought I am too fast with conclusions :)

Comment (pietrushnic) [reply docs/dev-proc/hcl-maintainer.md:26]: @miczyg1 let's try to go to a conclusion, what is your expectation in this conversation? If we can get some consensus with other reviewers on your expectations, I will apply those changes. I would like to move whole this topic further at some point.

@macpijan @sulewskiprzemyslaw for over two months, I would appreciate the feedback.

Comment (miczyg1) [reply docs/dev-proc/hcl-maintainer.md:26]: @pietrushnic if we say we dislike something, we should a least also say how it should look like so that we will like it. Few questions that arise from this short compliant:

```
Current template for adding HCL report over Github issue is way too complex.
Nobody will wast time doing that.
```

1. What makes current template way too complex?
2. Maybe some sections of the templates are redundant and are making it overly complex? If yes which ones?
3. What are the criteria of non-complex template?

Comment (pietrushnic) [reply docs/dev-proc/hcl-maintainer.md:26]: > What makes current template way too complex?

You have to provide up to [20 data points (10 required, 10 optional)](https://github.com/Dasharo/dasharo-issues/blob/master/.github/ISSUE_TEMPLATE/hcl-repor,t.md)
- according to [this book](https:/ninewww.amazon.com/Checklist-Manifesto-How-Things-Right/dp/0312430000) most extended procedure for pilots has 9 steps because research shows that prolonged procedures lead to rapid growth in mistakes; IIRC on average procedures have 4-5 pints
- there is no explanation if it is an easy, automatic way of gathering information
- **nobody ever filled even one such report**
- format of providing that information is unknown;

> Maybe some sections of the templates are redundant and are making it overly complex? If yes which ones?

This is already explained in the section proposing a new look for tables. There is a very minimal amount of information needed and if one wants the link to the source is provided (e.g. [CPU](https://github.com/Dasharo/docs/pull/463/files#diff-475b421b7f7e689cac83c3872d82d0319ba246ba1d11737b6a1b676ed17bd861R43)).

IMHO there is no use for existing HCL reporting using GH template, this method exist for 9+ months and there no even a single HCL report provided.

> What are the criteria of non-complex template?

Complexity is not a key problem, an efficient solution is what we aim for. It is just a hypothesis about why we have a problem. We only aim for efficient HCL reporting and maintenance on our side. If complexity leads to better efficiency then let it be.

At minimum what is presented in the Checklist Manifesto book would be helpful—ideally fully automated reporting instead of manual reports on GH. 

To sum up, I want to achieve efficient HCL report gathering and processing. Right now, we need better efficiency, and the goal of this PR is to improve efficiency. If it solves the discussion, I will remove the HCL report template and dismiss claims about complexity.

Please express what are you expecting and if you know a better way to address a key problem that this PR tries to solve.

Comment (miczyg1) [reply docs/dev-proc/hcl-maintainer.md:26]: > You have to provide up to [20 data points (10 required, 10 optional)](https://github.com/Dasharo/dasharo-issues/blob/master/.github/ISSUE_TEMPLATE/hcl-repor,t.md)

Ohh so you meant that one particular issue template, I see now what you meant.

> IMHO there is no use for existing HCL reporting using GH template, this method exist for 9+ months and there no even a single HCL report provided.

True. But we can only guess the reasons why there is not a single report yet added. Secondly, these are just templates that should help the community supply the information we need, they serve as a guidance, I have never treated it as fixed format. Although I agree, much more efficient approach would be good in some cases (e.g. drop a pack of logs instead of filling 10-20 options, however still certain data cannot be extracted from logs only - I can think of at least GPU exact model name, total VRAM, etc.).

> To sum up, I want to achieve efficient HCL report gathering and processing. Right now, we need better efficiency, and the goal of this PR is to improve efficiency. If it solves the discussion, I will remove the HCL report template and dismiss claims about complexity.

Well, I think we can remove it. But I still have some concerns about consuming the efficient HCL reports on our side. By consuming I mean extracting the value we need. Let's take two hypothetical users who submit HCL reports:

1. Regular user, who submits a bunch of logs and we have to extract value from it by ourselves.
2. Enthusiats user who experiments, sends us logs and screenshots, but also describe the results and summarizes it.

With type 1 we cannot expect the person to provide much more additional data even if we need it (most likely it will be to difficult for that person).

With type 2 there is high chance w can get almost any info we will eventually need. Furthermore type 2 reports have significant more value, much of the hard work is already done by the person and what is important we have incredible input for knowledge base and documentation.

We have a mass of type 1 reports on cloud pushed via DTS right? Are we doing something with them? Nope, almost nothing. It makes some use in extremely rare cases (returning the firmware dumps t their original owners excluded). In general, it also contains a configuration that is well tested by us.

While we have only a handful number of type 2 reports:
- https://github.com/Dasharo/dasharo-issues/issues/173
- another big one incoming from memory experiments from Dasharo IRC

They bring much more value, have much more sophisticated hardware configurations, reveal more bugs, often with clear reasons of their occurrence, and huge, invaluable amount of knowledge.

So my main question is now, where do we look for efficiency? On our side? or on the user side submitting such report?

Comment (pietrushnic) [reply docs/dev-proc/hcl-maintainer.md:26]: > Although I agree, much more efficient approach would be good in some cases (e.g. drop a pack of logs instead of filling 10-20 options, however still certain data cannot be extracted from logs only - I can think of at least GPU exact model name, total VRAM, etc.).

IMO if we cannot get data automatically we should not gather it. There is too much problem with data gathered that way.

> With type 2 there is high chance w can get almost any info we will eventually need. Furthermore type 2 reports have significant more value, much of the hard work is already done by the person and what is important we have incredible input for knowledge base and documentation.

I'm not sure how this comment affect whole HCL report efficiency topic.

> We have a mass of type 1 reports on cloud pushed via DTS right? Are we doing something with them? Nope, almost nothing. It makes some use in extremely rare cases (returning the firmware dumps t their original owners excluded). In general, it also contains a configuration that is well tested by us.

@miczyg1 this PR (and related work) is about that. There are also internal tasks regarding HCL (DTS improvements, Dasharo Transparent Validation improvements) which coming from this PR. This PR is the place to change way we treat HCL reports ideally make it automatic and close to zero manual effort. What I propose here are scripts which automatically generate tables based on HCL reports - those table can be copy pasted to Dasharo Universe documentation. This is very far from ideal but without discussion and further work on the topic not much will change.

> They bring much more value, have much more sophisticated hardware configurations, reveal more bugs, often with clear reasons of their occurrence, and huge, invaluable amount of knowledge.

Great. I'm not sure how it contradicts anything I propose in PR. It is even better we probably can use gh cli to pull files and extract value from packages to feed Dasharo Universe HCL pages.

> So my main question is now, where do we look for efficiency? On our side? or on the user side submitting such report?

Both, whole process is flawed. HCL report from DTS works pretty well what we can see by number of packages on our cloud. Key issue is that we have value sitting there which we do not process. More to that SEO shows Dasharo Universe benefit from compatibility lists because many people look for such information. TBH we should have HCL reports for all supported hardware.

File: docs/variants/msi_z690/cpu-hcl.md [hunk 2]
```diff
@@ -2,41 +2,158 @@
...
-[Dasharo documentation repository](https://github.com/Dasharo/docs) or by using
-[Dasharo issues repository](https://github.com/Dasharo/dasharo-issues/issues).
+* Use [Dasharo Tools Suite HCL
+  report](../../../dasharo-tools-suite/documentation/#hcl-report) to upload
+  report automatically.
+* Create new issue in [Dasharo issues
+  repository](https://github.com/Dasharo/dasharo-issues/issues).
+* Create PR directly to [Dasharo documentation
+  repository](https://github.com/Dasharo/docs).
+* [email](mailto:contact@dasharo.com?subject=CPU HCL report) you
```
[the remark this answers is not in the document]
Comment (macpijan) [reply docs/variants/msi_z690/cpu-hcl.md:12]: Maybe add hardware model to subject as well?

File: docs/variants/msi_z690/cpu-hcl.md [hunk 3]
```diff
@@ -2,41 +2,158 @@
...
 ## Contributing
 
-Feel free to test different processors and report your results via
-[email](mailto:contact@dasharo.com) or submit a Pull Request to
-[Dasharo documentation repository](https://github.com/Dasharo/docs) or by using
-[Dasharo issues repository](https://github.com/Dasharo/dasharo-issues/issues).
+* Use [Dasharo Tools Suite HCL
+  report](../../../dasharo-tools-suite/documentation/#hcl-report) to upload
+  report automatically.
+* Create new issue in [Dasharo issues
```
[the remark this answers is not in the document]
Comment (macpijan) [reply docs/variants/msi_z690/cpu-hcl.md:8]: We can also suggest issue title by adding `&title=` to the URL, similarly as with e-mail subject

File: docs/variants/msi_z690/cpu-hcl.md [hunk 4]
```diff
@@ -2,41 +2,158 @@
...
+=== "CPU HCL"
+
+    CPU Hardware Compatibility List presents the CPUs tested and verified
+    to work with Dasharo by community. The following list does not include CPU
+    which is tested and verfied in 3mdeb laboratory - this information might be
+    found in [Hardware Matrix](hardware-matrix.md) documentation.
+
+    Legend:
+
+    * CPU Model: Intel CPU Model name.
```
Comment (macpijan) [inline docs/variants/msi_z690/cpu-hcl.md:34]:
```suggestion
    * CPU Model: CPU Model name.
```

File: docs/variants/msi_z690/cpu-hcl.md [hunk 5]
```diff
@@ -2,41 +2,158 @@
...
+    * Dasharo version: Dasharo version on which report was created.
+        - `-` if version was not reported
+    * Source:
+        - Link to report if it is public.
+        - `Dasharo HCL report` if it was reported using DTS.
```
Comment (macpijan) [inline docs/variants/msi_z690/cpu-hcl.md:39]: We may link to https://docs.dasharo.com/dasharo-tools-suite/documentation/#hcl-report so someone new can find more information about that

File: docs/variants/msi_z690/cpu-hcl.md [hunk 6]
```diff
@@ -2,41 +2,158 @@
...
+    | Kingston | KF3600C17D4/8GX      | 8192 MB | DDR4 SDRAM | 2400 MT/s (PC4-19200) | &#10004/-/&#10004 | v1.0.0 | Dasharo HCL report |
+    | Kingston | KF3600C17D4/8GX      | 8192 MB | DDR4 SDRAM | 2400 MT/s (PC4-19200) | -/-/&#10004 | v1.1.0 | Dasharo HCL report |
+    | Patriot Memory | 4400 C19 Series | 8192 MB | DDR4 SDRAM | 2133 MT/s (PC4-17000) | &#10004/-/- | v1.0.0 | Dasharo HCL report |
+    | Thermaltake Technology Co Ltd | RG26D408GX2-3600C18A | 8192 MB | DDR4 SDRAM | 2666 MT/s (PC4-21300) | &#10004/-/- | v1.1.0 | Dasharo HCL report |
+
+    1) Tested in 2x2 mixed configuration
+
+### HCL list - MSI PRO Z690-A DDR5
+
+| DIMM vendor | Part Number | Size | SPD profile | Configuration 1/2/4 |
```
Comment (macpijan) [inline docs/variants/msi_z690/cpu-hcl.md:108]: This table is less complete than the previous one

File: docs/variants/msi_z690/cpu-hcl.md [hunk 7]
```diff
@@ -2,41 +2,158 @@
...
+[5]: https://www.kingston.com/datasheets/KF436C18BBK2_64.pdf
+[6]: https://www.kingston.com/memory/gaming/kingston-fury-beast-ddr5-memory
+[7]: https://www.crucial.com/memory/ddr5/ct2k16g48c40u5
+[8]: https://www.kingston.com/datasheets/HX426C16FRK4_64.pdf
+[9]: https://www.adata.com/us/consumer/dram-module-ddr5-4800-u-dimm
+[10]: https://www.amazon.com/OLOy-Memory-288-Pin-Desktop-MD4U323216DJDA/dp/B0836NZHWR
+
+=== "GPU HCL"
+
+    GPU Hardware Compatibility List presents the GPUs tested and verified
```
Comment (macpijan) [inline docs/variants/msi_z690/cpu-hcl.md:127]: `tested and verified` may mean a lot. In the DDR section we explain a bit what do we mean by that.

File: docs/variants/msi_z690/cpu-hcl.md [hunk 8]
```diff
@@ -2,41 +2,158 @@
...
+    * Memory size: total amount of GPU memory declared by vendor.
+    * Memory type: GPU's type of memory.
+    * Bandwidth: GPU's bandwidth.
+    * PCI-E Architecture: declared by producer generation of PCI-E architecture.
+    * Multi-Graphics Technology: information about support for Multi-Graphics
+        Technology.
+
+    Information about GPU might be read from GPU package or documentation.
+
+    | GPU name         | Memory size | Memory type  | Bandwidth | PCI-E Gen | Multi-Graphics Technology | Results                |
```
Comment (macpijan) [inline docs/variants/msi_z690/cpu-hcl.md:144]: In prevoius table it is `Source`, not `Results`

File: docs/variants/msi_z690/cpu-hcl.md [hunk 9]
```diff
@@ -2,41 +2,158 @@
...
+    |:----------------:|:-----------:|:------------:|:---------:|:---------:|:-------------------------:|:----------------------:|
+    | Nvidia GeForce GTX 1060   | 3072 MB  | GDDR5  | 192GB/s   | Gen3      | 1                         | [Qubes HCL reports][1] |
+    | MSI Radeon RX 6950 XT     | 16 GB    | GDDR6  | 576GB/s   | Gen4      | 1                         | |
+    | EVGA NVidia RTX 2080      | 8 GB     | GDDR6  | 448GB/s   | Gen3      | 1                         | |
+    | PNY NVidia RTX A5000      | 24 GB    | GDDR6  | 768GB/s   | Gen4      | 1                         | |
+    | Nvidia GeForce GTX 1080TI | 11264 MB | GDDR5X | x16       | Gen3      | 1                         | [Qubes HCL reports][2] |
+    | MSI Radeon RX 6500 XT MECH 2X 4G OC | 4 GB     | GDDR6  | x4 (x16 connector) | Gen4      | 1                         | Works only on Dasharo v1.1.0 or newer |
```
Comment (macpijan) [inline docs/variants/msi_z690/cpu-hcl.md:151]: Why `Works only on Dasharo v1.1.0 or newer` is in the column, where source of reports was supposed to be like? Version information is supposed to be in a dedidacted column, as with previous components. It is missing for GPU.

File: docs/variants/msi_z690/cpu-hcl.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pietrushnic) [reply docs/variants/msi_z690/cpu-hcl.md]: Typo fixed: https://github.com/Dasharo/docs/pull/463/commits/a1cdccd111fab4339e37d0beec4740bf8620a939

> Maybe add hardware model to subject as well?

Like this? https://github.com/Dasharo/docs/pull/463/commits/ac3711e512317b85ce089cdc44f54b897490b43a

File: docs/variants/msi_z690/cpu-hcl.md [hunk 10]
```diff
@@ -2,41 +2,158 @@
...
+  report automatically.
+* Create new issue in [Dasharo issues
+  repository](https://github.com/Dasharo/dasharo-issues/issues).
+* Create PR directly to [Dasharo documentation
+  repository](https://github.com/Dasharo/docs).
+* [email](mailto:contact@dasharo.com?subject=CPU HCL report for msi_z690) your
+  `/proc/cpuinfo` or relevant information using following template:
+
+  ```text
+  CPU model:
```
Comment (macpijan) [inline docs/variants/msi_z690/cpu-hcl.md:16]: How about contributing  to DDR and GPU lists?

File: docs/variants/msi_z690/cpu-hcl.md [hunk 11]
```diff
@@ -2,41 +2,158 @@
 
 ## Contributing
 
-Feel free to test different processors and report your results via
-[email](mailto:contact@dasharo.com) or submit a Pull Request to
-[Dasharo documentation repository](https://github.com/Dasharo/docs) or by using
-[Dasharo issues repository](https://github.com/Dasharo/dasharo-issues/issues).
+* Use [Dasharo Tools Suite HCL
```
Comment (macpijan) [inline docs/variants/msi_z690/cpu-hcl.md:5]: Maybe we need to rename the file, if it contains more than just CPU HCL?

File: docs/variants/msi_z690/cpu-hcl.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (macpijan) [reply docs/variants/msi_z690/cpu-hcl.md]: Right now we will not confuse it with Z790 at least, for example. I am not sure how WiFi/non-WiFi DDR4/DDR5 is important here. It is a good start.

File: docs/variants/msi_z690/cpu-hcl.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (pietrushnic) [reply docs/variants/msi_z690/cpu-hcl.md:8]: We can only address some of those suggestions, and we should decide what would be best.

The link was improved here: https://github.com/Dasharo/docs/pull/463/commits/3b09c4e70a6d6c134d5c2307b3870fd7adafddbb.

I used the same title for the email, but I disagree with the whole concept of HCL reporting over GH. If we had separate small HCL reports for CPU/GPU/Memory, it would work, but I opt for minimizing user effort and maximally automating the work of HCL generation through CI.

File: docs/variants/msi_z690/cpu-hcl.md (hunk 5 above)
Comment (pietrushnic) [reply docs/variants/msi_z690/cpu-hcl.md]: We already link to it in Contributing section, but I fixed it anyway: https://github.com/Dasharo/docs/pull/463/commits/514e3ce2d40fc8f007b7eb62b312826352d7a693

File: docs/variants/msi_z690/cpu-hcl.md (hunk 6 above)
Comment (pietrushnic) [reply docs/variants/msi_z690/cpu-hcl.md:109]: @macpijan it is; the whole work still needs to be completed. It is more of RFC; before investing too much time, I wanted to get your feedback about the change in approach. Please note that it took us three months to check this PR - it isn't efficient. Let's say I will redesign the whole concept of reporting all components. It still has to align with all other efforts of DTS and TAT otherwise, we keep having 3 competing solutions.  I had hoped for some practical cross-team cooperation.

**Comment by linuxhw** [comment]
Several Dasharo devices are already collected in the database: https://linux-hardware.org/?view=search&name=Dasharo#list

**Comment by pietrushnic** [comment]
@linuxhw thanks for letting us know. We saw those statistics and wondered how to improve the SMBIOS table to clarify things in both hardware databases. We also considered Linux Hardware and BSD Hardware as reference points for configuration reporting, meaning the extension of [DTS](https://docs.dasharo.com/dasharo-tools-suite/overview/). It would be great to discuss how to build synergy correctly.

@macpijan please let me know what are your expectations, it seems that you accept the approach presented in the CPU description, but I would like to make sure that the whole team is on the same page. The number of reports is growing, and we could expose even more useful information to end users - I would expect that to happen before `DUG#2`. If my approach is approved and all other parties in 3mdeb will comply, then I can continue development for GPU and memory.

File: docs/dev-proc/hcl-maintainer.md (hunk 1 above)
Comment (pietrushnic) [reply docs/dev-proc/hcl-maintainer.md:26]: @miczyg1 ping

Comment (pietrushnic) [reply docs/dev-proc/hcl-maintainer.md:27]: @miczyg1 ping2

File: docs/variants/msi_z690/cpu-hcl.md (hunk 6 above)
Comment (pietrushnic) [reply docs/variants/msi_z690/cpu-hcl.md:135]: @macpijan it looks like we don't have HCL reports dumps for DDR5 because those were not generated during testing. So either we leave this table as is, despite we have no proof in the form of logs or we remove it.

File: docs/variants/msi_z690/cpu-hcl.md [hunk 12]
```diff
@@ -2,41 +2,158 @@
...
+
+    | Manufacturer | Part Number | Size | Type | Speed | Configuration 1/2/4 | Dasharo version | Source |
+    |:------------:|:-----------:|:----:|:----:|:-----:|:-------------------:|:---------------:|:------:|
+    | Crucial Technology | CT8G4DFS8266.C8FJ | 8192 MB | DDR4 SDRAM | 2666 MT/s (PC4-21300) | &#10004/-/- | v1.1.0 | Dasharo HCL report |
+    | Kingston | KF3600C17D4/8GX      | 8192 MB | DDR4 SDRAM | 2400 MT/s (PC4-19200) | &#10004/-/&#10004 | v1.0.0 | Dasharo HCL report |
+    | Kingston | KF3600C17D4/8GX      | 8192 MB | DDR4 SDRAM | 2400 MT/s (PC4-19200) | -/-/&#10004 | v1.1.0 | Dasharo HCL report |
+    | Patriot Memory | 4400 C19 Series | 8192 MB | DDR4 SDRAM | 2133 MT/s (PC4-17000) | &#10004/-/- | v1.0.0 | Dasharo HCL report |
+    | Thermaltake Technology Co Ltd | RG26D408GX2-3600C18A | 8192 MB | DDR4 SDRAM | 2666 MT/s (PC4-21300) | &#10004/-/- | v1.1.0 | Dasharo HCL report |
+
+    1) Tested in 2x2 mixed configuration
```
[the remark this answers is not in the document]
Comment (pietrushnic) [reply docs/variants/msi_z690/cpu-hcl.md]: This part was entirely overwritten by new CPU HCL results.

File: docs/variants/msi_z690/cpu-hcl.md (hunk 7 above)
Comment (pietrushnic) [reply docs/variants/msi_z690/cpu-hcl.md:154]: @macpijan Accurate, and in the context of GPU, it is not easy to claim about `tested and verified`. What do you suggest? Please note those are not my words. Those are words contributed by @sulewskiprzemyslaw approved and verified by us: https://github.com/Dasharo/docs/pull/208/files

File: docs/variants/msi_z690/cpu-hcl.md (hunk 8 above)
Comment (pietrushnic) [reply docs/variants/msi_z690/cpu-hcl.md:187]: @macpijan here is my take on GPU HCL: https://github.com/Dasharo/docs/pull/463/commits/53e4c86f8a0ee2446862d7913a74dfc12a174c6f

Please let me know WDYT

File: docs/variants/msi_z690/cpu-hcl.md (hunk 10 above)
Comment (pietrushnic) [reply docs/variants/msi_z690/cpu-hcl.md:15]: @macpijan added in https://github.com/Dasharo/docs/pull/463/commits/87c3fcc07b81eb29f05171ee8392a9fbcd9dbc33

File: docs/variants/msi_z690/cpu-hcl.md (hunk 11 above)
Comment (pietrushnic) [reply docs/variants/msi_z690/cpu-hcl.md]: @macpijan fixed: https://github.com/Dasharo/docs/pull/463/commits/b15568a7f22156f1106be853b27ad78feecbcd94

**Comment by pietrushnic** [comment]
@macpijan @miczyg1 For the last three months, I didn't get any feedback from you. There is not much interest and not much harm this PR can do, so I proceed with merging. We should apply this new method of HCL maintenance for new Dasharo-supported hardware.

**Comment by miczyg1** [comment]
> @macpijan @miczyg1 For the last three months, I didn't get any feedback from you. There is not much interest and not much harm this PR can do, so I proceed with merging. We should apply this new method of HCL maintenance for new Dasharo-supported hardware.

I already left some essays in the comments earlier. As I completely don't feel the change of HCL format, I don't see much value in putting even more energy into it. If you merged it, that's fine. To me HCL is neither good nor bad, there will always be someone saying "I miss this info or that info here" or "this info is useless", etc. on a whim.

**Comment by pietrushnic** [comment]
@miczyg1 I sent a ping twice, but I understand that those changes don't care for you and @macpijan. That's why I just merged those. The scalability of the previous approach was, IMHO, not acceptable. Now I would kindly ask to comply with the new HCL layout for new hardware platforms. That's it.
